addons/web_editor/static/src/js/backend/convert_inline.js

```diff
@@ -337,6 +337,9 @@ function cardToTable($editable) {
             superCol.append(subTable);
             superRow.append(superCol);
             table.append(superRow);
+            if (child.nodeType !== Node.ELEMENT_NODE) {
+                continue;
+            }
             const hasImgTop = [child, ...child.querySelectorAll('.card-img-top')].some(node => (
                 node.classList && node.classList.contains('card-img-top') && node.closest && node.closest('.card') === table
             ));
```

Comment:
Good catch! Just a style suggestion because that's the style favored in the module and it makes things more explicit.

```suggestion
            if (child.nodeType === Node.ELEMENT_NODE) {
                const hasImgTop = [child, ...child.querySelectorAll('.card-img-top')].some(node => (
                    node.classList &&
                    node.classList.contains('card-img-top') &&
                    node.closest && node.closest('.card') === table
                ));
```
... and then the closing `}` after the other `if` (GitHub won't let me comment this far).